CONTRIBUTING.md

```diff
@@ -49,32 +56,46 @@ Subscribe to all the MQTT topics the `yggd` client will publish and subscribe to
 to monitor the MQTT traffic.
 
 ```
-sub -broker tcp://test.mosquitto.org:8883 -topic yggdrasil/$(hostname)/#
+sub -broker tcp://localhost:1883 -topic yggdrasil/$(hostname)/#
 ```
 
 ### Terminal 4
 
 Publish a "PING" command to the `yggd` "control/in" topic.
 
 ```
-go run ./cmd/yggctl generate control-message --type command '{"command":"ping"}' | pub -broker tcp://test.mosquitto.org:8883 -topic yggdrasil/$(hostname)/control/in
+go run ./cmd/yggctl generate control-message --type command '{"command":"ping"}' | \
+    pub -broker tcp://localhost:1883 -topic yggdrasil/$(hostname)/control/in
 ```
 
 You should see the message logged to the output of `sub` in [Terminal
 3](#terminal-3) and receipt of the message logged in the output of `yggd` in
-[Terminal 1](#terminal-1).
+[Terminal 1](#terminal-1). The `echo` worker does not receive any message in this
+case.
 
-Now publish a data message to the echo worker.
+Now publish a data message "hello" to the `echo` worker.
 
 ```
-go run ./cmd/yggctl generate data-message --directive worker "hello" | pub -broker tcp://test.mosquitto.org:8883 -topic yggdrasil/$(hostname)/data/in
+go run ./cmd/yggctl generate data-message --directive echo "hello" | \
+    pub -broker tcp://localhost:1883 -topic yggdrasil/$(hostname)/data/in
 ```
 
+> Note: if you implemented your own worker (e.g. called cool_worker),
+> you will need to replace the `echo` directive with the name of your
+> worker (--directive cool_worker).
+
```

Comment:
These doesn't need to be part of the quickstart. This knowledge will come later when the reader begins to explore what a worker is and how to communicate with them. Not all readers will be worker developers. There is also a [Worker Developer Guide](https://github.com/RedHatInsights/yggdrasil/wiki/Worker-Developer-Guide) on the wiki.